src/components/editor/EditorPane.tsx

```diff
@@ -60,6 +60,7 @@ function diff(a: any[], b: any[]): { ab: any[], ba: any[] } {
 export class EditorPane extends React.Component<EditorPaneProps, {
   views: Map<File, View>;
 }> {
+  editor: Editor;
```

Comment:
I think this is accidentally left over. (I wonder if our linting tools can be configured to detect this.)